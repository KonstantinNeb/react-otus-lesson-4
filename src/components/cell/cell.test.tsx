import React from 'react';
import { render } from '@testing-library/react';
import Cell from './cell';

describe("simple snapshot tests", () => {
  it("Default cell snapshot", () => {
    const { container } = render(<Cell />)
    expect(container).toMatchSnapshot()
  })

  it("nonDefaultID cell snapshot", () => {
    const { container } = render(<Cell id={123} />)
    expect(container).toMatchSnapshot()
  })
})