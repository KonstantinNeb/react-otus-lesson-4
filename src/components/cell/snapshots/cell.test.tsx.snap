// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple snapshot tests Default cell snapshot 1`] = `
<div>
  <div
    class="Cell"
  >
    <div
      class="Cell-div-parent"
    >
      <div
        class="Cell-div-child"
        id="0"
      >
        0
      </div>
    </div>
  </div>
</div>
`;

exports[`simple snapshot tests nonDefaultID cell snapshot 1`] = `
<div>
  <div
    class="Cell"
  >
    <div
      class="Cell-div-parent"
    >
      <div
        class="Cell-div-child"
        id="123"
      >
        123
      </div>
    </div>
  </div>
</div>
`;